src/Servers/Kestrel/Transport.Quic/src/WebHostBuilderQuicExtensions.cs

```diff
@@ -9,10 +9,15 @@
 namespace Microsoft.AspNetCore.Hosting;
 
 /// <summary>
-/// Quic <see cref="IWebHostBuilder"/> extensions.
+/// <see cref="IWebHostBuilder" /> extension methods to configure the Quic transport to be used by Kestrel.
 /// </summary>
 public static class WebHostBuilderQuicExtensions
 {
+    /// <summary>
+    /// Specify Quic as the transport to be used by Kestrel.
+    /// </summary>
+    /// <param name="hostBuilder">The <see cref="IWebHostBuilder"/>. to configure.</param>
+    /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
```

Comment:
💭 Maybe make this directly reference the parameter to show that it's a specific instance and not just an arbitrary instance of the type?

```suggestion
    /// <returns>Returns <paramref name="hostBuilder"/>.</returns>
```